### PR TITLE
Added new Components(basic version)

### PR DESCRIPTION
**Pull request by Akido123**
Ingredients.js is exception Ingredients object as props and instruction.js is expecting instructions object as props

### CODE REVIEW

File: components/description/ingredients.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Fragment } from "react";
```
Comment (Kenneth-Ngcobo) [inline components/description/ingredients.js:1]: I don't like using old school fragment, please use empty tag fragment

**Review by Kenneth-Ngcobo — CHANGES_REQUESTED** [review]
Please remove old Fragment

**Review by kgakatsikeamogetswe — DISMISSED** [review]
I have reviewed

File: components/description/ingredients.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function Ingredients(props){
+  const { ingredients } = props;
+
+  return(
+    <>
+      {ingredients.map((ingredient) => {
```
Comment (Kenneth-Ngcobo) [inline components/description/ingredients.js:6]: You need a return() after opening the map function brackets {}
Perhaps to test if your component does display data, copy data objects from Mongo Compass, pass them as props to your components

File: components/description/instructions.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function Instructions(props){
+  const { instructions } = props
+
+  return(
+    <>
+      {instructions.map((instruction, index) => {
```
Comment (Kenneth-Ngcobo) [inline components/description/instructions.js:6]: You need a return() after opening the map function brackets {}
Perhaps to test if your component does display data, copy data objects from Mongo Compass, pass them as props to your components

**Review by Kenneth-Ngcobo — CHANGES_REQUESTED** [review]
The map function doesn't have implicit return if there are multiple items in the function body. You need to put a return inside it. 
Copy a few objects from Mongo Compass, use them as dummy data to test if your components are functioning correctly

**Pull request closed without merging**